RecoMuon/Configuration/python/RecoMuonPPonly_cff.py

```diff
@@ -88,6 +88,9 @@
 run3_GEM.toModify( standAloneMuons, STATrajBuilderParameters = dict(
     FilterParameters = _enableGEMMeasurement, 
     BWFilterParameters = _enableGEMMeasurement ) )
```

Comment:
era modifiers should be applied where the modules are defined.
In this case, in RecoMuon/StandAloneMuonProducer/python/standAloneMuons_cfi.py

Actually, once that is done, modifications to displacedStandAloneMuons and refittedStandAloneMuons are not needed anymore because these are clones from standAloneMuons